moped-editor/src/views/projects/projectView/ProjectComponents/EditAttributesModal.js

```diff
@@ -134,14 +134,16 @@ const EditAttributesModal = ({
         subphase: makeSubphaseFormFieldValue(clickedComponent.moped_subphase),
         completionDate: clickedComponent.completion_date,
         srtsId:
-          clickedComponent.srts_id?.length > 0 ? clickedComponent.srts_id : "",
+          clickedComponent.srts_id?.length > 0
```

Comment:
I was able to save an empty string into the `description` column on update I'm pretty sure that the the initial value for `description` a few lines up needs this same code change. I tested it out and ran into a `!` on the `description` field in the `UPDATE_COMPONENT_ATTRIBUTES` mutation so that would need to be removed too to allow `null` to be passed. 🤦‍♂️ to me since I wrote that code. Thank you for looking at this. 🙏

Reply:
i cant get an empty string to save in the description column, but I was able to get `"     "` saved in the location_description column. 

Also ran into the issue where the description is required. I got thrown off by the mutation being named UpdateSignalComponent too. 
```
export const UPDATE_COMPONENT_ATTRIBUTES = gql`
  mutation UpdateSignalComponent(
    $projectComponentId: Int!
    $componentId: Int!
    $description: String!
```

Reply:
thank you both! i did these things:
- made `description` correctly fallback to `null` instead of `""`
- made `description` nullable in the update mutation
- renamed the mutation from `UpdateSignalComponent` to `UpdateProjectComponent`